diff -aurN avahi-0.6.4.orig/avahi-python/avahi-discover.desktop.in avahi-0.6.4/avahi-python/avahi-discover.desktop.in
--- avahi-0.6.4.orig/avahi-python/avahi-discover.desktop.in	2005-11-15 01:21:50.000000000 +0100
+++ avahi-0.6.4/avahi-python/avahi-discover.desktop.in	2006-01-22 19:52:51.417051000 +0100
@@ -2,9 +2,9 @@
 Encoding=UTF-8
 Name=Avahi Zeroconf Browser
 Comment=Browse for Zeroconf services available on your network
-Exec=@bindir@/avahi-discover
+Exec=avahi-discover
 Terminal=false
 Type=Application
-Icon=gnome-nettool.png
-Categories=GNOME;Application;System;Utility
+Icon=avahi.png
+Categories=GTK;System;Utility
 StartupNotify=false
