--- avahi-0.6.24/avahi-python/avahi-discover/avahi-discover.desktop.in.in~	2008-12-12 21:46:47.000000000 +0100
+++ avahi-0.6.24/avahi-python/avahi-discover/avahi-discover.desktop.in.in	2008-12-16 14:26:56.000000000 +0100
@@ -3,10 +3,10 @@
 Encoding=UTF-8
 _Name=Avahi Zeroconf Browser
 _Comment=Browse for Zeroconf services available on your network
-Exec=@bindir@/avahi-discover
+Exec=avahi-discover
 Terminal=false
 Type=Application
-Icon=network-wired
-Categories=GNOME;Application;System;
+Icon=avahi
+Categories=GTK;System;Utility;
 StartupNotify=false
 GenericName=
