--- avahi-0.6.21/avahi-common/dbus-watch-glue.c.orig	2007-08-12 17:39:21.000000000 +0200
+++ avahi-0.6.21/avahi-common/dbus-watch-glue.c	2007-08-27 22:12:48.357476786 +0200
@@ -134,7 +134,7 @@
 
         if (!(avahi_watch = poll_api->watch_new(
                   poll_api,
-                  dbus_watch_get_unix_fd(dbus_watch),
+                  dbus_watch_get_fd(dbus_watch),
                   translate_dbus_to_avahi(dbus_watch_get_flags(dbus_watch)),
                   watch_callback,
                   dbus_watch)))
