--- avahi-0.6.26/avahi-autoipd/Makefile.am~	2010-06-28 18:23:15.000000000 +0200
+++ avahi-0.6.26/avahi-autoipd/Makefile.am	2010-07-07 11:15:32.407583823 +0200
@@ -58,8 +58,6 @@
 	$(AM_V_GEN)cp $< $@
 endif
 
-if TARGET_DEBIAN
-
 noinst_SCRIPTS = dhclient-enter-hook dhclient-exit-hook
 
 dhclient-enter-hook: dhclient-enter-hook.in
@@ -72,8 +70,8 @@
 
 BUILD = dhclient-exit-hook dhclient-enter-hook
 
-dhcliententerdir = $(sysconfdir)/dhcp/dhclient-enter-hooks.d
-dhclientexitdir = $(sysconfdir)/dhcp/dhclient-exit-hooks.d
+dhcliententerdir = $(sysconfdir)/dhclient-enter-hooks.d
+dhclientexitdir = $(sysconfdir)/dhclient-exit-hooks.d
 
 install-exec-hook: dhclient-exit-hook dhclient-enter-hook
 	$(mkdir_p) $(DESTDIR)$(dhcliententerdir) $(DESTDIR)$(dhclientexitdir)
@@ -83,7 +81,6 @@
 uninstall-hook:
 	rm -f $(DESTDIR)$(dhcliententerdir)/avahi-autoipd $(DESTDIR)$(dhclientexitdir)/avahi-autoipd
 
-endif
 
 
 endif
