Index: avahi-core/netlink.c
===================================================================
--- avahi-core/netlink.c	(revision 1329)
+++ avahi-core/netlink.c	(working copy)
@@ -62,6 +62,12 @@
 
     p = (struct nlmsghdr *) nl->buffer;
     
+    /* Check that this message originated from the kernel,
+       or a request from avahi itself, and not another process */
+    if ((p->nlmsg_pid != 0) && (p->nlmsg_pid != getpid())) {
+        return -1;
+    }
+
     assert(nl->callback);
     
     for (; bytes > 0; p = NLMSG_NEXT(p, bytes)) {
