--- avahi-0.8/avahi-python/avahi-discover/avahi-discover.desktop.in.in.orig	2020-02-26 20:34:56.989881924 +0100
+++ avahi-0.8/avahi-python/avahi-discover/avahi-discover.desktop.in.in	2020-02-26 20:39:31.478394892 +0100
@@ -2,9 +2,9 @@
 Version=1.0
 Name=Avahi Zeroconf Browser
 Comment=Browse for Zeroconf services available on your network
-Exec=@bindir@/avahi-discover
+Exec=avahi-discover
 Terminal=false
 Type=Application
-Icon=network-wired
-Categories=GNOME;System;
+Icon=avahi
+Categories=GTK;System;Utility;Network;
 StartupNotify=false
