--- avahi-0.7/avahi-autoipd/avahi-autoipd.action.linux~	2017-07-10 12:12:38.000000000 +0200
+++ avahi-0.7/avahi-autoipd/avahi-autoipd.action.linux	2017-10-26 09:10:43.453751871 +0200
@@ -35,20 +35,20 @@ PATH="$PATH:/usr/bin:/usr/sbin:/bin:/sbi
 
 METRIC=$((1000 + `cat "/sys/class/net/$2/ifindex" 2>/dev/null || echo 0`))
 
-if [ -x /bin/ip -o -x /sbin/ip ] ; then
+if [ -x /sbin/ip ] ; then
 
     # We have the Linux ip tool from the iproute package
 
     case "$1" in
         BIND)
-            ip addr flush dev "$2" label "$2:avahi"
-            ip addr add "$3"/16 brd 169.254.255.255 label "$2:avahi" scope link dev "$2"
-            ip route add default dev "$2" metric "$METRIC" scope link ||:
+            /sbin/ip addr flush dev "$2" label "$2:avahi"
+            /sbin/ip addr add "$3"/16 brd 169.254.255.255 label "$2:avahi" scope link dev "$2"
+            /sbin/ip route add default dev "$2" metric "$METRIC" scope link ||:
             ;;
 
         CONFLICT|UNBIND|STOP)
-            ip route del default dev "$2" metric "$METRIC" scope link ||:
-            ip addr del "$3"/16 brd 169.254.255.255 label "$2:avahi" scope link dev "$2"
+            /sbin/ip route del default dev "$2" metric "$METRIC" scope link ||:
+            /sbin/ip addr del "$3"/16 brd 169.254.255.255 label "$2:avahi" scope link dev "$2"
             ;;
 
         *)
