diff -urN avahi-0.6.19/avahi-ui-sharp/Makefile.am avahi-0.6.19.new/avahi-ui-sharp/Makefile.am
--- avahi-0.6.19/avahi-ui-sharp/Makefile.am	2007-04-21 20:47:16.000000000 +0200
+++ avahi-0.6.19.new/avahi-ui-sharp/Makefile.am	2007-06-18 17:27:08.498654208 +0200
@@ -61,10 +61,10 @@
 endif
 
 install-data-hook: $(ASSEMBLY)
-	$(GACUTIL) /i $(ASSEMBLY) /package avahi-ui-sharp /gacdir $(libdir) /root $(DESTDIR)$(libdir)
+	$(GACUTIL) /i $(ASSEMBLY) /package avahi-ui-sharp /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib
 
 uninstall-hook: $(ASSEMBLY)
-	$(GACUTIL) /u avahi-ui-sharp /package avahi-ui-sharp /gacdir $(libdir) /root $(DESTDIR)$(libdir)
+	$(GACUTIL) /u avahi-ui-sharp /package avahi-ui-sharp /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib
 
 endif
 endif
--- avahi-0.6.26/avahi-sharp/Makefile.am~	2010-06-25 20:45:23.000000000 +0200
+++ avahi-0.6.26/avahi-sharp/Makefile.am	2010-07-07 11:12:57.330099139 +0200
@@ -73,10 +73,10 @@
 endif
 
 install-data-hook: $(ASSEMBLY)
-	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /i $(ASSEMBLY) /package avahi-sharp /gacdir $(libdir) /root $(DESTDIR)$(libdir)
+	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /i $(ASSEMBLY) /package avahi-sharp /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib
 
 uninstall-hook: $(ASSEMBLY)
-	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /u avahi-sharp /package avahi-sharp /gacdir $(libdir) /root $(DESTDIR)$(libdir)
+	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /u avahi-sharp /package avahi-sharp /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib
 
 endif
 endif
