--- avahi-0.6.25/avahi-python/avahi-discover/avahi-discover.desktop.in.in~	2009-04-15 09:55:26.000000000 +0200
+++ avahi-0.6.25/avahi-python/avahi-discover/avahi-discover.desktop.in.in	2009-04-15 09:57:18.000000000 +0200
@@ -2,10 +2,10 @@
 Version=1.0
 _Name=Avahi Zeroconf Browser
 _Comment=Browse for Zeroconf services available on your network
-Exec=@bindir@/avahi-discover
+Exec=avahi-discover
 Terminal=false
 Type=Application
-Icon=network-wired
-Categories=GNOME;System;
+Icon=avahi
+Categories=GTK;System;Utility;
 StartupNotify=false
 GenericName=
