diff -ru avahi-0.6.26.old/configure.ac avahi-0.6.26/configure.ac
--- avahi-0.6.26.old/configure.ac	2010-06-29 02:30:32.000000000 +0200
+++ avahi-0.6.26/configure.ac	2010-07-09 14:34:57.396332889 +0200
@@ -564,7 +564,7 @@
 fi
 AM_CONDITIONAL(HAVE_GTK3, test "x$HAVE_GTK3" = "xyes")
 
-AM_CONDITIONAL(HAVE_GTK2OR3, test "x$HAVE_GTK3" = "xyes" -o "x$HAVE_GTK2" = "xyes" )
+AM_CONDITIONAL(HAVE_GTK2OR3, test "x$HAVE_GTK3" = "xyes" -o "x$HAVE_GTK" = "xyes" )
 
 #
 # D-Bus
