--- avahi-0.6.26/avahi-gobject/AvahiCore-0.6.gir.wiget	2010-07-09 23:18:45.296015778 +0200
+++ avahi-0.6.26/avahi-gobject/AvahiCore-0.6.gir	2010-07-09 23:19:00.748515016 +0200
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
             xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
