diff -urN avahi-0.6.32.org/avahi-daemon/main.c avahi-0.6.32/avahi-daemon/main.c
--- avahi-0.6.32.org/avahi-daemon/main.c	2016-02-16 03:13:45.000000000 +0100
+++ avahi-0.6.32/avahi-daemon/main.c	2016-05-27 01:47:37.741433179 +0200
@@ -1141,7 +1141,7 @@
     ignore_signal(SIGPIPE);
 
     if (!(nss_support = avahi_nss_support()))
-        avahi_log_warn("WARNING: No NSS support for mDNS detected, consider installing nss-mdns!");
+        avahi_log_warn("WARNING: No NSS support for mDNS detected, consider installing nss_mdns!");
 
     if (!(simple_poll_api = avahi_simple_poll_new())) {
         avahi_log_error("Failed to create main loop object.");
diff -urN avahi-0.6.32.org/man/avahi-bookmarks.1.xml.in avahi-0.6.32/man/avahi-bookmarks.1.xml.in
--- avahi-0.6.32.org/man/avahi-bookmarks.1.xml.in	2016-02-16 03:13:45.000000000 +0100
+++ avahi-0.6.32/man/avahi-bookmarks.1.xml.in	2016-05-27 01:47:37.738099757 +0200
@@ -53,7 +53,7 @@
         <optdesc><p>Create links pointing to mDNS host names instead
         of resolved IP addreses. This is only compatible with your
         browser if you run some kind of local NSS module to resolve
-        mDNS host names (e.g. nss-mdns). If both -A and -H are ommited
+        mDNS host names (e.g. nss_mdns). If both -A and -H are ommited
         avahi-bookmarks detects whether NSS support is available
         locally. This option conflicts with -A.</p></optdesc>
       </option>
diff -urN avahi-0.6.32.org/man/avahi-daemon.8.xml.in avahi-0.6.32/man/avahi-daemon.8.xml.in
--- avahi-0.6.32.org/man/avahi-daemon.8.xml.in	2016-02-16 03:13:45.000000000 +0100
+++ avahi-0.6.32/man/avahi-daemon.8.xml.in	2016-05-27 01:47:37.738099757 +0200
@@ -38,7 +38,7 @@
       use of the mDNS record cache the avahi-daemon maintains. First
       there is the so called "simple protocol" which is used
       exclusively by avahi-dnsconfd (a daemon which configures unicast
-      DNS servers using server info published via mDNS) and nss-mdns
+      DNS servers using server info published via mDNS) and nss_mdns
       (a libc NSS plugin, providing name resolution via mDNS). Finally
       there is the D-Bus interface which provides a rich object
       oriented interface to D-Bus enabled applications.</p>
